lit/docs/operation/creds.lit

```diff
@@ -46,3 +46,4 @@ backend you want to use.
 \include-section{creds/caching.lit}
 \include-section{creds/redacting.lit}
 \include-section{creds/retry.lit}
+\include-section{creds/cyberark-conjur.lit}
```

Comment:
![Screen Shot 2020-03-24 at 10 45 16 AM](https://user-images.githubusercontent.com/6372810/77439039-f29ee400-6dbc-11ea-8b9a-7e7bfa509759.png)

I think like this would be easier to navigate if all of the credential manager options were grouped together. Could we move *The Conjur credential manager* to item 1.11.2.6, and bump Caching, Redacting, and Retrying down? 

Then it would look like this:
**1.11.2 | Credential Management**
- 1.11.2.1 | The Vault credential manager
- 1.11.2.2 | The CredHub credential manager
- 1.11.2.3 | The AWS SSM credential manager
- 1.11.2.4 | The AWS Secrets Manager credential manager
- 1.11.2.5 | Kubernetes Credential Manager
- 1.11.2.6 | The Conjur credential manager
- 1.11.2.7 | Caching credentials
- 1.11.2.8 | Redacting credentials
- 1.11.2.9 | Retrying failed fetches

Reply:
This has been resolved https://github.com/concourse/docs/pull/291/commits/289aded037f490d3e54d6d99b36a191fc891ecc4